azure_pipelines/automation.yml

```diff
@@ -78,7 +78,6 @@ jobs:
         base64 --decode MSAL/test/automation/conf_encoded.json > MSAL/test/automation/conf.json
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->